test/client/js/libs/handlebars/parse.js

```diff
@@ -243,6 +243,49 @@ describe("parse Handlebars helper", () => {
 		expect(actual).to.deep.equal(expected);
 	});
 
+	it("should find nicks", () => {
+		const testCases = [{
+			users: ["MaxLeiter"],
+			input: "test, MaxLeiter",
+			expected:
+				"test, " +
+				"<span role=\"button\" class=\"user color-12\" data-name=\"MaxLeiter\">" +
```

Comment:
Can you use single quotes instead to avoid escaping? (See my PR I just merged)

Reply:
This was merged before yours. You should have enforced this with eslint rule by the way.